runtime/router/router.go

```diff
@@ -153,3 +158,14 @@ func (r *Router) allowed(path, reqMethod string) string {
 
 	return strings.Join(allow, ", ")
 }
+
+func (r *Router) isWhitelistedPath(path string) bool {
+	if len(r.WhitelistedPaths) > 0 {
+		for _, whitelistedPath := range r.WhitelistedPaths {
+			if strings.HasPrefix(path, whitelistedPath) {
+				return true
```

Comment:
check for equality rather than prefix

Reply:
We have to check prefix here because for _get_ method the /:var in the path can get changed but the prefix will remain same. To determine whether this is whitelisted path only prefix will tell that.

Reply:
Wouldn't a whitelisted path `/a/b/:var` match with an existing path `/a/b/` because the prefixes are the same then?

Reply:
In that case, we need do pattern matching for whitelisted paths. May be we need to create another trie for this?